iberian_labs.md

```diff
@@ -4,3 +4,90 @@ title: Iberian Labs
 ---
 
 If you would like your research group to be listed in our website, please fill in <a href="https://forms.gle/X3XvHLDNbY7jF1QX6" target="_blank">this form</a>.
+
+We will list only one entry per research institution/center (no duplicates).
+
+## Iberian Labs
+* ### Biomedical Imaging Laboratory
+Clínica Universidad de Navarra
+
+Pamplona, Spain
+
+The **Biomedical Imaging Laboratory** lab works in translational research projects, developing advanced MRI techniques and evaluating their use for disease diagnosis and characterization.
+
+Main research topics: *arterial spin labeling, renal imaging, myocardial perfusion, cerebrovascular reactivity*
+
+Contact: Maria A. Fernández Seara, mfseara@unav.es
```

Comment:
```suggestion
Contact: Maria A. Fernández Seara, [mfseara@unav.es](mailto:mfseara@unav.es)
```

Reply:
I think it would be helpful to have email accounts so that when you click on them your email app opens with the email accounts as a recipient.